presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -714,7 +714,9 @@ public void testApproxPercentile()
                 "   approx_percentile(orderkey, 0.5), " +
                 "   approx_percentile(totalprice, 0.5)," +
                 "   approx_percentile(orderkey, 2, 0.5)," +
-                "   approx_percentile(totalprice, 2, 0.5)\n" +
+                "   approx_percentile(totalprice, 2, 0.5)," +
```

Comment:
@martint error seems to be coming from here? I don't see anything wrong, any insight?

> [ERROR]   TestJdbcDistributedQueries>AbstractTestQueries.testApproxPercentile:712->AbstractTestQueryFramework.computeActual:113->AbstractTestQueryFramework.computeActual:118 Â» Runtime

Reply:
Looks like you found a bug in the QuantileDigest implementation. I have a fix for it: https://github.com/airlift/airlift/pull/736